Deprecates zksync Goerli and adds new zkSync Sepolia testnet

githubdoramon opened the pull request:
Also updates some minor information on all zkSync networks

ligi reviewed (changes requested):
If you deprecate a chain -> you MUST add the status deprecated field
If you really want to reactivate a deprecated chain -> you MUST add the redFlags reusedChainId info

githubdoramon commented:
> If you deprecate a chain -> you MUST add the status deprecated field If you really want to reactivate a deprecated chain -> you MUST add the redFlags reusedChainId info

Thanks @ligi . Addressed the comments

Romsters commented on _data/chains/eip155-300.json:
```diff
@@ -1,17 +1,30 @@
-  "name": "(deprecated) Optimism on GC",
-  "chain": "OGC",
-  "rpc": [],
+  "name": "zkSync Sepolia Mainnet",
```
```suggestion
  "name": "zkSync Sepolia Testnet",
```
?

githubdoramon replied:
oh god. Thanks

tilemanrenovations commented:
Hows it going for you Mr .  Digimon?

9844272456 commented:
ในวันที่ พฤ. 7 ธ.ค. 2023 06:54 น. Travis C. Nelson Jr. <
***@***.***> เขียนว่า:

> Hows it going for you Mr . Digimon?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ethereum-lists/chains/pull/3921#issuecomment-1843878091>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BEKQF246IEQTWB2BHCE4CBTYIEASJAVCNFSM6AAAAABAJBWKU6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTQNBTHA3TQMBZGE>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

githubdoramon commented:
@ligi friendly nudge :) - Can you please review again based on the changes?
